src/estimators/stratified_sketch.py

```diff
@@ -55,21 +107,41 @@ def __init__(self,
       random_seed: This arg exists in order to conform to
         simulator.EstimatorConfig.sketch_factory.
       cardinality_sketch_factory: A cardinality sketch factory.
```

Comment:
It looks like the parameters are out of order in the doc string.  Also, is noiser the same as noiser_class?  Need to define underlying_set.

Reply:
ah thanks, changed it so many times, forgot to update the doc string.